.cargo/config.toml

```diff
@@ -30,3 +30,9 @@ rustflags = [
 
 [net]
 git-fetch-with-cli = true
+
+
+[[patch.unused]]
```

Comment:
Whats the difference between this and the workaround format? I have not seen `patch.unused` before.

eg:
```
[patch.crates-io]
cc = { git = "https://github.com/rust-lang/cc-rs", rev = "e5bbdfa" }
```

Reply:
You're right. Good work on #33.